backend/src/email/utils/callback/update-status.ts

```diff
@@ -39,7 +39,7 @@ export const updateBouncedStatus = async (
     errorCode,
     errorSubType,
     timestamp: metadata.timestamp,
-    id: metadata.id,
```

Comment:
Similar to [this comment](https://github.com/opengovsg/postmangovsg/pull/1880/commits/d49222a7cb867138799b06916391106311e6e7d8#r1109204869), I think it's generally best to avoid using very generic variable names (like `id`) unless the context is super clear (i.e. "the ID of what?"). In most cases for this diff I felt it was clearer to qualify the variable name as `messageId`.